packages/core/components/Drawer/index.tsx

```diff
@@ -22,22 +27,35 @@ const DrawerDraggable = ({
   children,
   id,
   index,
+  type,
 }: {
   children: ReactNode;
   id: string;
   index: number;
-}) => (
-  <Draggable
-    key={id}
-    id={id}
-    index={index}
-    showShadow
-    disableAnimations
-    className={() => getClassNameItem()}
-  >
-    {() => children}
-  </Draggable>
-);
+  type: keyof DefaultComponentProps;
+}) => {
+  const appContext = useAppContext();
+
```

Comment:
@chrisvxd it's this code that is causing the docs site to fail build.  Any ideas why?

Reply:
I think that's because the Docs doesn't have the app context

Drawer is (sort of) designed to be general purpose and usable outside of the Puck context

I think I would move this logic out of Drawer and into the ComponentList, passing in `isDragDisabled` as a prop when using Drawer.